doc/whats_new.rst

```diff
@@ -127,6 +127,9 @@ API
 
     - EOG, ECG and EMG channels are now plotted by default (if present in data) when using :func:`mne.viz.plot_evoked` by `Marijn van Vliet`_
 
+
+    - Replace pseudoinverse-based solver with much faster Cholesky solver in :func:`mne.stats.regression.linear_regression_raw`, by `Jona Sassenhagen`_.
```

Comment:
FYI `linear_regression_raw` actually lives more readily accesible in the `mne.stats` namespace